src/OrbitBase/File.cpp

```diff
@@ -60,6 +63,17 @@ ssize_t pwrite(int fd, const void* buffer, size_t size, int64_t offset) {
 
 #endif
 
+namespace {
+
+template <typename TimePoint>
+std::time_t to_time_t(TimePoint time_point) {
```

Comment:
maybe add a todo to switch this to file_clock::to_sys() once we migrate to C++20.

Reply:
Done! Thanks for the suggestion!